packages/react-menu/Spec.md

```diff
@@ -1021,7 +1021,7 @@ Once the submenu is open, the same behavior as in the [previous section](#submen
 Keyboard interactions required to navigate the menu. The alphanumeric match interaction does not need to be supported in all cases, but should be supported as much as possible.
 
 | Type     | Action    | Result            | Details                                                               | Focus after                                |
-| -------- | --------- | ----------------- | --------------------------------------------------------------------- | ------------------------------------------ |
+| -------- | --------- | ----------------- | --------------------------------------------------------------------- | ------------------------------------------ | ---------------------- |
```

Comment:
I think this is related to line 1030 that has `### MenuItem selection` at the end. That probably should be in a new line.